Yubico.YubiKey/src/Yubico/YubiKey/Management/Commands/GetDeviceInfoCommand.cs

```diff
@@ -22,6 +23,7 @@ namespace Yubico.YubiKey.Management.Commands
     /// <remarks>
     /// This class has a corresponding partner class <see cref="GetDeviceInfoResponse"/>
     /// </remarks>
+    [Obsolete("This class has been replaced by GetPagedDeviceInfoCommand")]
```

Comment:
Does `nameof(GetPagedDeviceInfoCommand)` and interpolation work here? I know attribute data needs to be compile-time constants, but I can't remember if C# is smart enough to do that if all constituents are also const...

Reply:
Unfortunately no, it must be constant. Is it better to remove the reference to the new class and simply refer to the changelog?

Reply:
No - it's good to mention the class name. This string will be present in a compiler warning someone will see if they are using this class. So It's good to make those as descriptive as possible.